mkdocs.yml

```diff
@@ -13,7 +13,7 @@ site_dir: target/mkdocs
 # In that case subdirectories can be used to organize pages.
 # The list below organizes them into categories and controls the order.
 
-pages:
+nav:
```

Comment:
What is this change?

Reply:
The `pages` key has long been deprecated and renamed to `nav` but in the latest mkdocs version, which I installed to test the new doc theme, you have to actually use `nav`.